src/libcore/cmp.rs

```diff
@@ -401,6 +401,7 @@ pub fn max<T: Ord>(v1: T, v2: T) -> T {
 /// ```
 #[inline]
 #[unstable(feature = "cmp_partial")]
+#[deprecated(since = "1.3.0", reason = "has not proven itself worthwhile")]
```

Comment:
This only deprecates `partial_min`, but not `partial_max`. Are you sure that’s not an oversight?